packages/shared/src/hooks/useFeed.ts

```diff
@@ -77,6 +81,7 @@ export default function useFeed<T>(
   const { query, variables, options = {}, settings } = params;
   const { user, tokenRefreshed } = useContext(AuthContext);
   const queryClient = useQueryClient();
+  const isFeedPreview = feedQueryKey?.[0] === RequestKey.FeedPreview;
```

Comment:
Hmm I see here we don't have an alternative :O

Reply:
Feedname is not available everywhere so just wanted to have it the same in all places. This is a flaster either way until lean feed.

Reply:
Sure that's true :)